srcpkgs/sway/patches/primary_selection.diff

```diff
@@ -0,0 +1,18 @@
+--- sway/server.c
++++ sway/server.c
```

Comment:
Please explain the patch on the top (linking the PRs would work) and mention that the feature was only removed from wlroots after the 0.13 release, which is why we can still use it in sway.